src/ParallelAlgorithms/Amr/Actions/Actions.hpp

```diff
@@ -0,0 +1,10 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+/// \ingroup AmrGroup
+namespace amr {
```

Comment:
In `Amr.hpp` you have `\ingroup ComputationalDomainGroup`.  Probably one of these should be removed.

Reply:
agreed